includes/settings.php

```diff
@@ -454,3 +455,24 @@ function sanitize_settings( $settings ) {
 
 	return $new_settings;
 }
+
+/**
+ * Enable auto-update of plugin.
+ *
+ * @return void
+ */
+function auto_update_plugin() {
+	if ( ! DT_IS_NETWORK ) {
+		$valid_license = Utils\get_settings()['valid_license'];
+	} else {
+		$valid_license = Utils\get_network_settings()['valid_license'];
+	}
+
+	if ( $valid_license ) {
+		\Puc_v4_Factory::buildUpdateChecker(
```

Comment:
This class should always exist but we may want to add a `class_exists( 'Puc_v4_Factory' )` check here, as we do that same thing in other places